File: backend/src/main/java/me/performancereservation/domain/bookmark/Bookmark.java
package me.performancereservation.domain.bookmark;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import me.performancereservation.domain.common.BaseEntity;

@Getter
@Entity
@NoArgsConstructor
public class Bookmark extends BaseEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id; // 찜 ID

    private Long userId;         // (FK) 유저 ID

    private Long performanceId;  // (FK) 공연 ID

    @Builder
    public Bookmark(Long id, Long performanceId, Long userId) {
        this.id = id;
        this.performanceId = performanceId;
        this.userId = userId;
    }
}

File: backend/src/main/java/me/performancereservation/domain/bookmark/Bookmark.kt
package me.performancereservation.domain.bookmark

import jakarta.persistence.*
import lombok.Builder
import me.performancereservation.domain.common.BaseEntity

@Entity
class Bookmark(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long? = null,       // 찜 ID

    val userId: Long,           // (FK)사용자 ID
    val performanceId: Long     // (FK)공연 ID
) : BaseEntity()